README.adoc

```diff
@@ -220,7 +220,6 @@ Set the [hotspot=allprojects file=1]`basic configuration` for the project. Apply
 
 You can now develop the application and the different modules together in dev mode by using the Liberty Gradle plug-in. To learn more about how to use dev mode with multiple modules, check out the https://github.com/OpenLiberty/ci.gradle/blob/main/docs/dev.md#multiple-modules[Documentation^].
```

Comment:
https://github.com/OpenLiberty/ci.gradle/blob/main/docs/dev.md#multiple-modules --> This link does not work.

Reply:
also found by Ruilin and will be addressed by Arun